includes/footer.html

```diff
@@ -0,0 +1,63 @@
+<footer class="page-footer">
+    <div class="container">
+        <div class="row">
+            <div class="col-md-3">
+                <h5 class="white-text footer-title">coala</h5>
+                <ul>
+                    <li><a class="grey-text text-lighten-3" href="http://coala.io/docs">Documentation</a></li>
+                    <li><a class="grey-text text-lighten-3" href="http://coala.io/newcomer">Newcomer Docs</a></li>
+                    <li><a class="grey-text text-lighten-3" href="http://http://docs.coala.io/en/latest/Help/FAQ.html">FAQ</a></li>
+                    <li><a class="grey-text text-lighten-3" href="http://blog.coala.io/">Blog</a></li>
+                    <li><a class="grey-text text-lighten-3" href="https://gratipay.com/coala/">Donate</a></li>
+                </ul>
+            </div>
+            <div class="col-md-9">
+                <div class="row">
+                <div class="col-md-4">
+                    <div class="footer-small-title">Sponsored By</div>
+                    <div class="footer-sponsor-img center">
+                    <a href="http://appveyor.com"><img height=30 src="resources/images/av.png"/></a>
```

Comment:
appveyor sponsors server resources just as digitalocean so let's put them into the same category